src/content/guides/efk-stack-app-oidc/index.md

```diff
@@ -0,0 +1,365 @@
+What is missing?
+- [ ] Finish opendistro-security plugin
+    - [ ] Add overview
+    - [ ] Add explanation to `roles_key: groups` - this is what Kibana will use... 
+    - [ ] Identify other things that need explanation
+    - [ ] Explain them
+
+- [x] Add Configure internal-users database
+- [x] Add Configure roles-mapping to control users permission
+- [x] Add configuration of jwt token to send users group IDs
+- [x] Add explanation how azure AD roles are mapped to kibana roles
+
+---
+title: Configure oidc for efk-stack-app
+description: Configure authentication with kibana and the efk-stack-app through OIDC
+date: 2020-10-30
+type: page
+weight: 50
+tags: ["recipe"]
+---
+# Setup OIDC to authenticate with kibana and the efk-stack-app
+
+To leverage your companies existing user and group directories to manage access to the efk-stack-app the opendistro security plugin provides integration with different authentication backends.
+In this guid we will explain how to configure the efk-stack-app with Azure AD through the OpenID Connect (OIDC) standard. This configuration should be portable to any other authentication backend, that provides the OIDC standard.
+
+## Overview
```

Comment:
@ced0ps not sure if it's a big deal. But the headers don't align with the overview, which made it harder than it should be to follow. See: https://docs.google.com/document/d/1xVqXKDsmYMKV8SRHBbVLa5P16YQkhhkHDbRI0fIU7_g/edit?usp=sharing

Reply:
I implemented suggested changes here https://hackmd.io/UDcdTFJMSiGcbNK9kk-tFw